onnx/defs/rnn/defs.cc

```diff
@@ -5,7 +5,7 @@
 #include "onnx/defs/schema.h"
 
 namespace ONNX_NAMESPACE {
-void RNNShapeInference(InferenceContext& ctx) {
+static void RNNShapeInference(InferenceContext& ctx) {
```

Comment:
@cyyever @andife @gramalingam Would it make sense to expose operator-level shape inference functions through the ONNX_API to support contrib operators? This would enable users developing custom contrib ops to leverage existing shape inference implementations when their operators have similar semantics to standard ONNX ops, promoting code reuse and model consistency.